person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  attr_accessor :name, :age
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age?
+  end
```

Comment:
- [ ] Public method can_use_services? that returns true if person is of age or if they have permission from parents. in your case it is only the age. Lets add parent permission 😉 
```ruby
  def can_use_services? 
    of_age?() || @parent_permission
  end
```

- [ ] OPTIONAL: Usually private methods come at the end 😉